modules/tensor_mechanics/src/actions/GeneralizedPlaneStrainAction.C

```diff
@@ -41,12 +41,10 @@ GeneralizedPlaneStrainAction::act()
 {
   if (_current_task == "add_kernel")
   {
-    std::string k_type("GeneralizedPlaneStrainOffDiag");
+    std::string k_type = "GeneralizedPlaneStrainOffDiag";
     InputParameters params = _factory.getValidParams(k_type);
-
     params.set<std::vector<NonlinearVariableName> >("displacements") = _displacements;
-
-    params.set<std::vector<VariableName> >("scalar_strain_zz") = {_scalar_strain_zz};
+    params.set<std::vector<VariableName> >("scalar_strain_zz") = { _scalar_strain_zz };
```

Comment:
This is from personal communication with @permcody and @friedmud. The rationale is that we _have_ space on the interior of `{ }` in inline defined member functions!